### PR TITLE
Emit event when certificate gets updated

### PR DESCRIPTION
**Pull request by soltysh**
/assign @deads2k

### CODE REVIEW

File: pkg/operator/staticpod/certsyncpod/certsync_controller.go [hunk 1]
```diff
@@ -166,8 +167,10 @@ func (c *CertSyncController) sync() error {
 				continue
 			}
 		}
+		c.eventRecorder.Eventf("CertificateUpdated", "Writing updated configmap: %s/%s", configMap.Namespace, configMap.Name)
```
Comment (deads2k) [inline pkg/operator/staticpod/certsyncpod/certsync_controller.go]: handle the removal case above too.

File: pkg/operator/staticpod/certsyncpod/certsync_controller.go [hunk 2]
```diff
@@ -79,6 +79,7 @@ func getSecretDir(targetDir, secretName string) string {
 func (c *CertSyncController) sync() error {
 	errors := []error{}
 
+	klog.Infof("Syncing configmaps: %v", c.configMaps)
```
Comment (deads2k) [inline pkg/operator/staticpod/certsyncpod/certsync_controller.go]: do we have a good printer on these? I doubt it.

File: pkg/operator/staticpod/certsyncpod/certsync_controller.go [hunk 3]
```diff
@@ -256,6 +259,7 @@ func (c *CertSyncController) sync() error {
 				continue
 			}
 		}
+		c.eventRecorder.Eventf("CertificateUpdated", "Writing updated secret: %s/%s", secret.Namespace, secret.Name)
```
Comment (deads2k) [inline pkg/operator/staticpod/certsyncpod/certsync_controller.go]: removal case

File: pkg/operator/staticpod/certsyncpod/certsync_controller.go (hunk 1 above)
Comment (deads2k) [reply pkg/operator/staticpod/certsyncpod/certsync_controller.go]: emit events on errors from this I think.  We don't expect to see any.

File: pkg/operator/staticpod/certsyncpod/certsync_controller.go (hunk 2 above)
Comment (soltysh) [reply pkg/operator/staticpod/certsyncpod/certsync_controller.go]: You'll get name + optional pairs from the default writer, which imo is sufficient for now.

**Comment by soltysh** [comment]
@deads2k updated, ptal

**Comment by deads2k** [comment]
sure, let's see what happens.

/lgtm

@soltysh you may want to pick this back.

**Pull request merged**